adapters/emx_digital/usersync_test.go

```diff
@@ -27,9 +27,37 @@ func TestEMXDigitalSyncer(t *testing.T) {
 		},
 	})
 
+	// Validate TCFV1 consent string
+	assert.NoError(t, gdpr.ValidateConsent("BOPVK28OVJoTBABABAENBs-AAAAhuAKAANAAoACwAGgAPAAxAB0AHgAQAAiABOADkA"))
 	assert.NoError(t, err)
```

Comment:
Can we rename test to `TestEMXDigitalSyncerTCF1` to be consistent with the test below?